functions/src/index.ts

```diff
@@ -4,6 +4,7 @@ import * as eventService from './Controller/Event';
 import * as memberService from './Controller/Member';
 import * as committeeService from './Controller/Committee';
 import * as electionsService from './Controller/Elections';
+import * as positionService from './Controller/Position';
```

Comment:
hmmmmmmm. This might be kinda late for you guys but why is this called positionService instead of positionController if you're importing a controller

Reply:
Still don't have an answer for this.

Reply:
Makes sense, will update this soon.

Reply:
haniel, i think u and i called a service but i dont remember why

Reply:
but yea sebastian if u want to change go for it i guess